src/python-common/ceph/deployment/translate.py

```diff
@@ -0,0 +1,70 @@
+import logging
+from ceph.deployment.drive_group import DriveGroupSpec
+from ceph.deployment.drive_selection.selector import DriveSelection
+
+logger = logging.getLogger(__name__)
+
+
+class ToCephVolume(object):
```

Comment:
```suggestion
class to_ceph_volume(spec, seletion):
    ...
```

Having this as a function instead of a class make calling this a bit less awkward:

```python
translate.to_ceph_volume(spec, sel)
# vs
translate.ToCephVolume(spec, sel).run()
```